src/test/metric_config_test.cpp

```diff
@@ -433,7 +433,10 @@ TEST_F(ModelMetricReporterTest, MetricReporterConstructorTest) {
     ASSERT_TRUE(reporter5.requestFailGrpcGetModelMetadata != nullptr);
 }
 
-class MetricsCli : public ::testing::Test {};
+class MetricsCli : public ::testing::Test {
+public:
+    uint64_t restPort = 100000;
```

Comment:
allowed ports are in a range 1024-65536. It might be better to use lower port number in tests.

Reply:
Ok.